app/browser/dataset-browse.tsx

```diff
@@ -508,10 +532,10 @@ const NavSection = ({
 }: {
   label: React.ReactNode;
   icon: React.ReactNode;
-  items: (DataCubeTheme | DataCubeOrganization)[];
+  items: (DataCubeTheme | DataCubeOrganization | Termset)[];
```

Comment:
Nit: As we have `DataCube` prefixes for themes and organizations, would it make sense to have e.g. `DimensionTermset`? I think they are related to dimensions and not cubes, right?

Reply:
Not sure I want to go in this direction, I think I would even want to go in the other direction and have Theme and Organization be without prefix.